databases/sync_tests/test_batch.py

```diff
@@ -250,16 +250,16 @@ def test_execute_raw(client: Prisma, raw_queries: RawQueries) -> None:
     assert found.title == 'My new title'
 
 
-def test_create_many_unsupported(
+def test_skip_duplicates_unsupported(
     client: Prisma,
     config: DatabaseConfig,
 ) -> None:
-    """Cannot call create_many on databases that do not support it"""
-    if 'create_many' not in config.unsupported_features:
-        pytest.skip('The create_many method is supported by the current behaviour')
+    """Cannot call create_many with skip_duplicates on databases that do not support it"""
+    if 'skip_duplicates' not in config.unsupported_features:
+        pytest.skip('The create_many skip_duplicates argument is supported by the current behaviour')
```

Comment:
nit: this message was slightly wrong before 😅
```suggestion
        pytest.skip('The create_many skip_duplicates argument is supported by the current database')
```